pkg/repository/v1/sqlcv1/concurrency.sql

```diff
@@ -12,17 +12,18 @@ WHERE
 -- name: GetWorkflowConcurrencyQueueCounts :many
 SELECT
     w."name" AS "workflowName",
-    cs.key,
+    wcs.key,
     COUNT(*) AS "count"
 FROM
-    v1_concurrency_slot cs
+    v1_workflow_concurrency_slot wcs
```

Comment:
This is for rendering queue counts in the UI -- slightly more intuitive to show the workflow concurrency slots instead of the task concurrency slot counts